### PR TITLE
Update domain audit

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Update domain audit.

### Related issues (optional)
ENGDOCS-1157

[ENGDOCS-1157]: https://docker.atlassian.net/browse/ENGDOCS-1157?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docker-hub/domain-audit.md [hunk 1]
```diff
@@ -31,4 +33,8 @@ To audit your domains:
   - Email: The email address of the user.
   - Date Joined: The date the user created their Docker account.
```
Comment (BuraChuhadar) [inline docker-hub/domain-audit.md:34]: @craig-osterhout, the team will remove this column (Date Joined) from the feedback from our sales team. Can we please kindly remove this column?

**Review by ajthilakan — DISMISSED** [review]
lgtm

File: docker-hub/domain-audit.md [hunk 2]
```diff
@@ -29,6 +37,15 @@ To audit your domains:
   - Name: The name of the user.
   - Username: The Docker ID of the user.
   - Email: The email address of the user.
-  - Date Joined: The date the user created their Docker account.
 
-You can invite all the uncaptured users to your organization using the exported CSV file. For more details, see [Invite members via CSV file](../docker-hub/members.md/#invite-members-via-csv-file).
+You can invite all the uncaptured users to your organization using the exported CSV file. For more details, see [Invite members via CSV file](../docker-hub/members.md/#invite-members-via-csv-file). Optionally, enforce single sign-on or enable SCIM to automatically add users to your organization. For more details, see [Single Sign-on](../single-sign-on/index.md) or [SCIM](../docker-hub/scim.md).
```
Comment (jerae-duffin) [inline docker-hub/domain-audit.md:41]: Optionally, enforce single sign-on or enable SCIM to add users to your organization automatically.

File: docker-hub/domain-audit.md [hunk 3]
```diff
@@ -29,6 +37,15 @@ To audit your domains:
...
+
+> **Note**
+>
+> Domain audit may identify accounts of users who are no longer a part of your organization. If you don't want to add a user to your organization and you don't want the user to appear in future domain audits, the account must be deactivated or the associated email address must be updated.
```
Comment (jerae-duffin) [inline docker-hub/domain-audit.md:45]: If you don't want to add a user to your organization and you don't want the user to appear in future domain audits, you must deactivate the account or update the associated email address.

**Review by jerae-duffin — APPROVED** [review]
LGTM!

**Pull request merged**